samples/sample-vpc/template.yml

```diff
@@ -421,3 +391,146 @@ Resources:
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: 'ec2.amazonaws.com'
+            Action: 'sts:AssumeRole'
+      Policies:
+        - PolicyName: 'root'
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: 'Allow'
+                Action:
+                  - "logs:CreateLogGroup"
+                  - "logs:CreateLogStream"
+                  - "logs:DescribeLogGroups"
+                  - "logs:DescribeLogStreams"
+                  - "logs:PutLogEvents"
+                  - "logs:GetLogEvents"
+                  - "logs:FilterLogEvents"
+                Resource:
+                  - !Sub "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:${VPCFlogLogGroupName}/*"
+                  - !Sub "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:${VPCFlogLogGroupName}"
+
+Outputs:
+  VPCCidrRange:
+    Value:
+      !FindInMap ['SubnetConfig', 'VPC', 'CIDR']
+    Export:
+      Name: VPCCIDR
+
+  DefaultInternetConnectivitySecurityGroupId:
+    Description: The ID of the Internet connectivity security groups
+    Export:
+      Name:
+        Fn::Sub: ${Environment}-private-connectivity-security-group
+    Value:
+      Ref: DefaultInternetConnectivitySecurityGroup
+
+  PrivateRouteTableId:
+    Description: The ID of the Private Route Table
+    Export:
+      Name:
+        Fn::Sub: ${Environment}-private-route-table
+    Value:
+      Ref: PrivateRouteTable
+
+  PrivateSubnet1a:
+    Description: The ID of the Private Subnet 1a
+    Export:
+      Name:
+        Fn::Sub: ${Environment}-private-subnet-1a
+    Value:
+      Ref: PrivateSubnet1a
+
+  PrivateSubnet1b:
+    Description: The ID of the Private Subnet 1b
+    Export:
+      Name:
+        Fn::Sub: ${Environment}-private-subnet-1b
+    Value:
+      Ref: PrivateSubnet1b
+
+  PrivateSubnet1c:
+    Description: The ID of the Private Subnet 1c
+    Export:
+      Name:
+        Fn::Sub: ${Environment}-private-subnet-1c
+    Value:
+      Ref: PrivateSubnet1c
+
+  PublicRouteTableId:
+    Description: The ID of the Public Route Table
+    Export:
+      Name:
+        Fn::Sub: ${Environment}-public-route-table
+    Value:
+      Ref: PublicRouteTable
+
+  PublicSubnet1a:
+    Description: The ID of the Public Subnet 1a
+    Export:
+      Name:
+        Fn::Sub: ${Environment}-public-subnet-1a
+    Value:
+      Ref: PublicSubnet1a
+
+  PublicSubnet1b:
+    Description: The ID of the Public Subnet 1b
+    Export:
+      Name:
+        Fn::Sub: ${Environment}-public-subnet-1b
+    Value:
+      Ref: PublicSubnet1b
+
+  PublicSubnet1c:
+    Description: The ID of the Public Subnet 1c
+    Export:
+      Name:
+        Fn::Sub: ${Environment}-public-subnet-1c
+    Value:
+      Ref: PublicSubnet1c
+
+  VPC:
+    Description: The ID of the main VPC
+    Export:
+      Name:
+        Fn::Sub: ${Environment}-vpc-id
```

Comment:
Nit: we are mixing `!Sub` and `Fn::Sub` should we make it consistent? I sugges we use the short form (`!Sub`) everywhere as we are also using shortform for `!Ref` and `!GetAtt`